packages/builder-core/package.json

```diff
@@ -0,0 +1,136 @@
+  "engines": {
+    "node": ">=8.2.0"
+  },
+  "scripts": {
+    "lint:fix": "node ../../node_modules/prettier/bin-prettier.js --write \"src/**/*.{js,ts}\" \"test/**/*.{js,ts}\" && npm run lint-src -- --fix && npm run lint-tests -- --fix",
+    "lint": "npm run lint-src && npm run lint-tests",
+    "lint-tests": "node -r ts-node/register ../../node_modules/tslint/bin/tslint --project ./tsconfig.json",
+    "lint-src": "node -r ts-node/register ../../node_modules/tslint/bin/tslint --project src/tsconfig.json",
+    "test": "node ../../node_modules/mocha/bin/mocha",
+    "coverage": "node ../../node_modules/nyc/bin/nyc.js ../../node_modules/mocha/bin/mocha",
+    "build": "node ../../node_modules/typescript/bin/tsc --build src",
+    "build-test": "node ../../node_modules/typescript/bin/tsc --build .",
+    "clean": "node ../../node_modules/rimraf/bin.js builtin-tasks internal *.d.ts *.map *.js build-test tsconfig.tsbuildinfo build"
+  },
+  "files": [
+    "builtin-tasks/",
+    "internal/",
+    "sample-project/",
+    "src/",
+    "*.d.ts",
+    "*.d.ts.map",
+    "*.js",
+    "*.js.map",
+    "LICENSE",
+    "README.md",
+    "recommended-gitignore.txt",
+    "console.sol"
+  ],
+  "devDependencies": {
+    "@types/chai": "^4.2.0",
+    "@types/ci-info": "^2.0.0",
+    "@types/debug": "^4.1.4",
+    "@types/download": "^6.2.4",
+    "@types/find-up": "^2.1.1",
+    "@types/fs-extra": "^5.1.0",
+    "@types/glob": "^7.1.1",
+    "@types/lodash": "^4.14.123",
+    "@types/node-fetch": "^2.3.7",
+    "@types/qs": "^6.5.3",
+    "@types/semver": "^6.0.2",
+    "@types/sinon": "^7.5.1",
+    "@types/uuid": "^3.4.5",
+    "@types/ws": "^7.2.1",
+    "chai": "^4.2.0",
+    "sinon": "^9.0.0",
+    "time-require": "^0.1.2"
+  },
+  "dependencies": {
+    "@nomiclabs/ethereumjs-vm": "^4.1.1",
+    "@solidity-parser/parser": "^0.5.2",
+    "@types/bn.js": "^4.11.5",
+    "@types/lru-cache": "^5.1.0",
+    "abort-controller": "^3.0.0",
+    "ansi-escapes": "^4.3.0",
+    "chalk": "^2.4.2",
+    "ci-info": "^2.0.0",
+    "cross-env": "5.0.5",
+    "debug": "^4.1.1",
+    "deepmerge": "^2.1.0",
+    "download": "^7.1.0",
+    "enquirer": "^2.3.0",
+    "eth-sig-util": "^2.5.2",
+    "ethereum-cryptography": "^0.1.2",
+    "ethereumjs-abi": "^0.6.8",
+    "ethereumjs-account": "^3.0.0",
+    "ethereumjs-block": "^2.2.0",
+    "ethereumjs-common": "^1.3.2",
+    "ethereumjs-tx": "^2.1.1",
+    "ethereumjs-util": "^6.1.0",
+    "find-up": "^2.1.0",
+    "fp-ts": "1.19.3",
+    "fs-extra": "^7.0.1",
+    "glob": "^7.1.3",
+    "io-ts": "1.10.4",
+    "is-installed-globally": "^0.2.0",
+    "lodash": "^4.17.11",
+    "merkle-patricia-tree": "^3.0.0",
+    "mocha": "^7.1.2",
+    "node-fetch": "^2.6.0",
+    "qs": "^6.7.0",
+    "raw-body": "^2.4.1",
+    "semver": "^6.3.0",
+    "slash": "^3.0.0",
+    "solc": "0.6.8",
+    "source-map-support": "^0.5.13",
+    "ts-essentials": "^2.0.7",
+    "tsort": "0.0.1",
+    "uuid": "^3.3.2",
+    "ws": "^7.2.1"
+  },
```

Comment:
we need to remove all ethereum specific things (ethereum, solc, solidity, ws...)